endpoints/api_config.py

```diff
@@ -258,6 +258,17 @@ def _CheckLimitDefinitions(limit_definitions):
       _CheckType(ld.default_limit, int, 'limit_definition.default_limit')
 
 
+_VALID_API_NAME = re.compile('^[a-z][a-z0-9]{0,39}$')
+
+
+def _CheckApiName(name):
+  valid = (_VALID_API_NAME.match(name) is not None)
+  if not valid:
```

Comment:
nit: maybe compact to `if not _VALID_API_NAME.match(name)`?

Reply:
PEP8 [recommends](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) being explicit about checking if a value is None.